testAsset.props

```diff
@@ -1,3 +1,6 @@
 <Project ToolsVersion="14.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <Import Project="build/DependencyVersions.props" />
+  <PropertyGroup>
+    <NoPackageAnalysis>true</NoPackageAnalysis>
```

Comment:
I'm not sure what this does, but don't we want to ensure our test assets are "correct"?  That way we ensure we are testing like a customer would be using the product.

Reply:
We're building test asset nuget packages with stable versions targeting netcoreapp 2.1. It warns that our dependency is pre release.